app/templates/views/notifications/invalid_precompiled_letter.html

```diff
@@ -12,6 +12,6 @@ <h1 class="heading-large">Letter</h1>
       Provided as PDF on {{ created_at|format_datetime_short }}
     </p>
     <p class="notification-status-cancelled">
-      Validation failed – this isn’t a PDF file that Notify can read
+      Validation failed – this is not a PDF file that Notify can read
```

Comment:
Would this be better?
```suggestion
      Validation failed – Notify cannot read this PDF file
```

Reply:
Yes, definitely. For this PR I stuck to just replacing negative contractions. I wanted us to pick up the rewording of some of our validation errors as a separate thing, but we can make this change now.